pg_dump-to-s3.sh

```diff
@@ -27,7 +26,7 @@ for db in "${dbs[@]}"; do
     echo "Database $db is archived"
 done
 
-# Delere old files
-echo "Delete old backups";
-$DIR/s3-autodelete.sh $S3_PATH "$MAX_DAYS days"
+# Delete old files
+# echo "Delete old backups";
+# $DIR/s3-autodelete.sh $S3_PATH "$MAX_DAYS days"
```

Comment:
Can you further clarify as to why exactly we are disabling the deletion of the old backups.

Reply:
It is a task for another ticket, and I haven't tested this part of the scripts yet.

Reply:
So setting the max days to be "60" is in the other ticket, which is a configurable value, but that doesn't necessarily mean we should comment on this code. That ticket is also in this sprint, so I think it makes sense to test the auto deletion locally rather comment out this code and the uncomment it out later.

Reply:
@ankitbrana I removed the commented code and modified the autodelete scripts